lib/go-atscfg/parentdotconfig.go

```diff
@@ -1897,8 +1884,8 @@ func getProfileParentConfigParams(tcParentConfigParams []tc.Parameter) (map[stri
 func getServerParentConfigParams(server *Server, profileParentConfigParams map[string]map[string]string) map[string]string {
 	// We only need parent.config params, don't need all the params on the server
 	serverParams := map[string]string{}
-	if server.Profile == nil || *server.Profile != "" { // TODO warn/error if false? Servers requires profiles
-		for name, val := range profileParentConfigParams[*server.Profile] {
+	if server.Profiles == nil || (*server.Profiles)[0] != "" { // TODO warn/error if false? Servers requires profiles
+		for name, val := range profileParentConfigParams[(*server.Profiles)[0] {
```

Comment:
Extra `[` here is causing compilation failure

Reply:
This was 4 commits ago